extensions/amp-story/1.0/bookend/bookend-element.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Layout} from '../../../../src/layout';
+
+/** Class corresponding to the <amp-story-bookend> DOM element tag.  */
+export class AmpStoryBookend extends AMP.BaseElement {
+
```

Comment:
Can we do the loading of the bookend in this class?

Reply:
We can, it will just create some code duplication from amp-story-bookend.js to keep it backwards compatible.

Reply:
As discussed offline, I will be doing this on a separate PR.